pkg/cluster/describe.go

```diff
@@ -219,16 +219,35 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		cluster.MultiAZ(),
 	)
 
-	// secureboot
-	if cluster.GCP().Security().SecureBoot() {
-		fmt.Printf("SecureBoot:             %t\n", cluster.GCP().Security().SecureBoot())
+	// AWS-specific info
+	if cluster.CloudProvider().ID() == ProviderAWS {
+		fmt.Printf("PrivateLink:		%t\n"+
+			"STS:			%t\n"+
+			"Subnet IDs:		%s\n",
+			privateLinkEnabled,
+			stsEnabled,
+			cluster.AWS().SubnetIDs(),
+		)
+	}
+
+	// GCP-specific info
+	if cluster.CloudProvider().ID() == ProviderGCP {
+		if cluster.GCP().Security().SecureBoot() {
+			fmt.Printf("SecureBoot:             %t\n", cluster.GCP().Security().SecureBoot())
+		}
+		if cluster.GCPNetwork().VPCName() != "" {
+			fmt.Printf("VPC-Name:	        %s\n", cluster.GCPNetwork().VPCName())
+		}
+		if cluster.GCPNetwork().ControlPlaneSubnet() != "" {
+			fmt.Printf("Control-Plane-Subnet:   %s\n", cluster.GCPNetwork().ControlPlaneSubnet())
+		}
+		if cluster.GCPNetwork().ComputeSubnet() != "" {
+			fmt.Printf("Compute-Subnet:	        %s\n", cluster.GCPNetwork().ComputeSubnet())
+		}
 	}
 
 	fmt.Printf("CCS:			%t\n"+
 		"HCP:			%t\n"+
```

Comment:
Wouldn't HCP also be AWS specific? At least for now

Reply:
I'll leave things as is for now since HCP could at some point be GCP-supported